### PR TITLE
Query and Update animals table

### PR DESCRIPTION
**Pull request by edubew**
## This milestone fulfills the following requirements:
- [x] Add a column species of type string to your animals table. Modify your schema.sql file.
- [x] Insert data into the animals table and modify data.sql file
- [x] Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
- [x] Inside a transaction delete all records in the animals table, then roll back the transaction.
- [x] After the rollback verify if all records in the animals table still exists.
- [x] Write queries to show the use of aggregate functions

**Query to update animals table inside transactions screenshot**
image

**Query to delete, restore and update data screenshots**
image
image
image

**Query with aggregate functions screenshots**
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,54 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
 SELECT name FROM animals WHERE NOT (name = 'Gabumon');
- SELECT name, weight_kg FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+SELECT name, weight_kg FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Query to update animals table inside transactions
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT name FROM animals WHERE species = 'unspecified';
+ROLLBACK;
```
Comment (alexander16108) [inline queries.sql:17]: as required please verify that the changes was done after the rollback 👍🏾

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,54 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+SELECT * FROM animals;
```
Comment (alexander16108) [inline queries.sql]: as required please verify that the changes was done before the commit to avoid errors in the future 👍🏾

File: queries.sql [hunk 3]
```diff
@@ -7,4 +7,54 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+
+-- Query to delete, restore and update data in animals table inside transactions
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```
Comment (alexander16108) [inline queries.sql]: as required please verify that the changes was done before the `rollback` to avoid errors in the future 👍🏾

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @edubew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
screenshots added ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @edubew , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your PR looks profesional. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**